### ISSUE TITLE
Wrong for calculation

### ISSUE THREAD

**Issue by ginesmartinezros** [zacharee/CameraXInfo#5]
Hi there!
I found your app and your database and I think it's really useful, thanks!
I'm interested in fov data, so I checked the calculation formula used
In `Utils.kt`, we have
`2f * atan(frameSize.run { height * 16.0 / 9.0 } / (focal * 2f)) * 180 / PI`
I think that you are assuming that the aspect ratio is 16:9. However, normally in camera sensors it's 4:3. This leads to a wrong fov calculations. I have a setup ready to make empirical fov measures and I have ensured this.
Therefore there are mainly two solutions to provide a correct FOV
* Use `width` instead of `height`
* Use actual aspectRatio which is generally 4:3
Besides, I have implemented an Excel formula to correct this fov accordingly and it works
`180/PI()*2*ATAN((width/height)/(16/9)*TAN(provided fov*PI()/180/2))`

**Comment by zacharee** [comment]
Good catch. I'm not really sure why I calculated it that way, honestly. I've switched to using the width in 1.3.0.
